Project-II-Calculator/calculator/src/App.js

```diff
@@ -1,23 +1,138 @@
 import React from 'react';
 import './App.css';
+import ActionButton from './components/ButtonComponents/ActionButton.js';
+import NumberButton from './components/ButtonComponents/NumberButton.js';
+import CalculatorDisplay from './components/DisplayComponents/CalculatorDisplay.js';
 
+const nums = [1,2,3,4,5,6,7,8,9];
+let total = 0;
+let formula = "";
+let displayFormula = "";
+const NumberPanel = props =>{
+    return props.function;
+  }
+
+/*
 const App = () => {
   return (
-    <div>
-      <h3>Welcome to React Calculator</h3>
-      <p>
-        We have given you a starter project. You'll want to build out your
-        components in their respective files, remove this code and replace it
-        with the proper components.
-      </p>
-      <p>
-        <strong>
-          Don't forget to `default export` your components and import them here
-          inside of this file in order to make them work.
-        </strong>
-      </p>
+    <div className = "calculator">
```

Comment:
all of this clogs up code that is otherwise clean and easy to read. Get into the habit of removing extraneous or unused code it can create accidental bugs.